src/metabase/analytics/stats.clj

```diff
@@ -355,15 +355,20 @@
 
 (defn- execution-metrics-sql []
   (let [thirty-days-ago (case (db/db-type)
-                          :postgres "CURRENT_TIMESTAMP - INTERVAL '30 days'"
+                          :postgres "CURRENT_TIMESTAMP AT TIME ZONE 'UTC' - INTERVAL '30 days'"
```

Comment:
I think it's worth an block comment in the code about why this is needed. Looking up the blame for an explanation is tedious, and the reasoning becomes even more obscure once code gets moved or reformatted.

Reply:
Sure, I'll add one